rules/linux/credential_access_potential_linux_ssh_bruteforce_internal.toml

```diff
@@ -48,14 +48,16 @@ The rule identifies consecutive SSH login failures targeting a user account from
 
 """
 risk_score = 47
-rule_id = "1c27fa22-7727-4dd3-81c0-de6da5555feb"
+rule_id = "a8fb5071-c071-4ec0-9ecd-85b45244ddfc"
```

Comment:
Is this going to cause any problems? It's sort of like deprecating a rule.

Reply:
Is this needed?

Reply:
@imays11 @w0rk3r I reverted the rule_id change. Back when I created this PR, I still thought that this was necessary when making changes to rules. Good catch.